.github/workflows/ci.yml

```diff
@@ -1,32 +1,50 @@
-name: Unit Tests 
-
-# Controls when the action will run. Triggers the workflow on push or pull request
-# events but only for the master branch
-on: 
-  pull_request:
+name: Unit Tests
+on:
   push:
-  workflow_dispatch:
-
```

Comment:
Does it make sense to keep `workflow_dispatch`? In the past was useful for debuggin the actions.